src/github/github.module.ts

```diff
@@ -0,0 +1,11 @@
+import { Module } from '@nestjs/common';
+import { GithubService } from './github.service';
+import { GithubApi } from './github.api';
+import { ConfigModule, ConfigService } from '@nestjs/config';
+
+@Module({
+  imports: [ConfigModule],
+  providers: [ConfigService, GithubService, GithubApi],
```

Comment:
```suggestion
  providers: [GithubService, GithubApi],
```

I don't think you need to mark the ConfigService in the providers, given you import the configured module